### PR TITLE
Change "Distincts" text to "distinct values"

### LINKED ISSUE #16035: Change 'Distinct' to 'Count Distinct' in graphical query builder
**Is your feature request related to a problem? Please describe.**

The fact that `Distinct` does what `count distinct` does in SQL is confusing: I strongly expected `Distinct` to produce a list of distinct values rather than a count of the number of distinct values, both when choosing it in a Summarize pulldown and (more strongly) when using the function in a custom expression.

**Describe the solution you'd like**

Change the name from `Distinct` to `Count Distinct` or some variation. (I realize this is a breaking change...)

**Describe alternatives you've considered**

Documentation is there, but the name `Distinct` is so suggestive that I assumed I was doing something wrong when it gave me a count instead.

**How important is this feature to you?**

Annoyance-level.

Related #805

### PR DESCRIPTION
**Pull request by daltojohnso**
Resolves #16035

1. Updated text
2. Updated tests dependent on text

Screen Shot 2022-04-29 at 5 00 37 PM

### CODE REVIEW

**Comment by daltojohnso** [comment]
@mazameli or @kdoh is this desirable?

File: frontend/src/metabase/modes/components/drill/SummarizeColumnDrill.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const AGGREGATIONS = {
   distinct: {
     section: "sum",
     buttonType: "token",
-    title: t`Distincts`,
+    title: t`Count Distinct`,
```
Comment (mazameli) [inline frontend/src/metabase/modes/components/drill/SummarizeColumnDrill.js:23]:
```suggestion
    title: t`Count distinct values`,
```

**Comment by daltojohnso** [comment]
Screen Shot 2022-04-29 at 5 29 48 PM

**Review by mazameli — APPROVED** [review]
I changed it to `Distinct values` with a capital D.

**Comment by mazameli** [comment]
Let's not backport this one into 0.43.0, @daltojohnso, since it changes a string. We can ship it in 0.43.1 if I can update strings again for that release.

**Pull request merged**